index.html

```diff
@@ -0,0 +1,609 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TNW</title>
+    <link rel="stylesheet" href="styles/reset.css">
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="icon" href="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/icons/favicon-194x194.png?v=1583748939">
+</head>
+
+<body>
+   <header>
+     <div class="navbar">
```

Comment:
[OPTIONAL]

- [ ] I strongly recommend changing the `<div>` tag on line 15 to be a `<nav>` tag

Reply:
Done